src/v1/destinations/linkedin_ads/networkHandler.js

```diff
@@ -46,7 +47,7 @@ const responseHandler = (responseParams) => {
           [tags.TAG_NAMES.ERROR_TYPE]: getDynamicErrorType(finalStatus),
         },
         destinationResponse,
-        getAuthErrCategoryFromStCode(status),
+        getAuthErrCategoryFromStCode(destinationResponse),
```

Comment:
Attach a sample destination response for clarity

Reply:
added